reinforcement_learning/common/sagemaker_rl/tf_serving_utils.py

```diff
@@ -1,9 +1,14 @@
 import ray
 import os
 import re
-from ray.rllib.utils.framework import try_import_tf
+import tensorflow as tf
 
-tf = try_import_tf()
+def import_tf1():
+    if "2." in tf.__version__[:2]:
+        return tf.compat.v1
+    return tf
+
+tf = import_tf1()
```

Comment:
how is this different from ray's native `try_import_tf`? That method also tries to switch off v2_behavior.

Reply:
That method is only available in later versions of Ray, and not in 0.6.5.

Reply:
We might need some more comments here to explain the context. I think we just switching from `tf` to `try_import_tf` in this commit: https://github.com/aws/amazon-sagemaker-examples/commit/44d0ffa9d2e90b90fa64fbc8e987012fe4b02250#diff-68432ce762a5889a6722a6ff6184996fd732f6cf8a9d6ca00b0d449f21ef5522

Reply:
1) I thought we are targeting ray 1.0 soon/going forward. so try_import_tf is safe here?